### ISSUE TITLE
Cannot create nested arrays.

### ISSUE THREAD

**Issue by Riobe** [json-schema-faker/json-schema-faker#282]
Description
==
http://json-schema-faker.js.org/#gist/b02a603d2b83004c7c67510b05ce216a/0.5.0-rc5 shows that when one of the items in an array type property is itself an array, the items are put into the parent array instead of the sub-array.

Expected:
==

```json
{
  "nestedArrays": [
    [
      -32420639,
      53102032,
    ],
    "dol"
  ]
}
```

Actual:
==

```json
{
  "nestedArrays": [
    -32420639,
    53102032,
    "dol"
  ]
}
```

**Comment by pateketrueke** [comment]
Nice one, actually this did never worked... let me think about it.

Thanks for your feedback!

**Comment by pateketrueke** [comment]
Yeah, the `.call` change just did the trick!

**Comment by pateketrueke** [comment]
Fixed on `0.5.0-rc6`, closing.
